### ISSUE TITLE
[Site Request] rumble.com

### ISSUE THREAD

**Issue by shakeyourbunny** [pukkandan/yt-dlp#80]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running yt-dlp version **2021.02.09**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs
https://rumble.com/vdmum1-moose-the-dog-helps-girls-dig-a-snow-fort.html
https://rumble.com/vdq1k5-uaes-hope-probe-enters-orbit-of-mars.html
https://rumble.com/vd17ez-apples-macbook-pro-redesign-in-2021.html

- Single video: https://rumble.com/vdih93-baby-humpback-whale.html
- Single video: https://rumble.com/vch7d1-stuff-of-genius-douglas-engelbart-the-computer-mouse.html
- Playlist: site has no playlists

## Description

Rumble is a video sharing platform that allows video content creators to upload their videos and share in the ad revenue received.  There are no requirements for subscribers or overall view time making it possible to earn ad revenue very quickly.  Rumble provides four different licensing options to give you greater control over your potential earnings.  If you choose the Exclusive Video Management license option and your video gets approved Rumble takes care of the distribution.

see also https://www.trippinbrehenys.com/2020/02/13/rumble-review/

**Comment by pukkandan** [comment]
Related: https://github.com/ytdl-org/youtube-dl/issues/27146 https://github.com/ytdl-org/youtube-dl/issues/27294 https://github.com/ytdl-org/youtube-dl/issues/27828 https://github.com/ytdl-org/youtube-dl/issues/10785

**Comment by pukkandan** [comment]
The embed URLs already work
```
$ youtube-dlc -F "https://rumble.com/embed/vb0ofn/?pub=4"
[RumbleEmbed] vb0ofn: Downloading JSON metadata
[info] Available formats for vb0ofn:
ID        EXT  RESOLUTION |   TBR PROTO | VCODEC    VBR ACODEC   ABR
--------- ---- ---------- - ----- ----- - ------- ----- ------- ----
mp4-360p  mp4  360p       |  635k https | unknown  635k unknown   0k
webm-480p webm 480p       |  811k https | unknown  811k unknown   0k
mp4-480p  mp4  480p       |  814k https | unknown  814k unknown   0k
mp4-720p  mp4  720p       | 1956k https | unknown 1956k unknown   0k
mp4-1080p mp4  1080p      | 2688k https | unknown 2688k unknown   0k
```

So the webpage URL will automatically start working once #12 is complete

**Comment by pukkandan** [comment]
I have implemented a temporary fix till #12 is complete
